hash/a7800.xml

```diff
@@ -977,6 +1032,19 @@ type of rom dumps.  Effort will be made at a later date to split them and docume
 		</part>
 	</software>
 
+	<software name="froggie">
+		<description>Froggie</description>
+		<year>2018</year>
+		<publisher>AtariAge</publisher>
```

Comment:
Is the correct spelling `Atari Age` or `AtariAge`? I have a xegs.xml entry that is spelled the other way around and I have no idea about what's the correct one.

Reply:
> Is the correct spelling `Atari Age` or `AtariAge`? I have a xegs.xml entry that is spelled the other way around and I have no idea about what's the correct one.

AtariAge is the site's proper name. No space.